Source/AlienRace/AlienRace/Graphic/Graphic.cs

```diff
@@ -0,0 +1,253 @@
+﻿using RimWorld;
+using UnityEngine;
+using Verse;
+namespace AvaliMod
+{
+    public class AvaliGraphic : Graphic

```

Comment:
The simplest way to make the third color work in graphics was just to create these new classes based of vanilla ones.

Reply:
In most spots, Graphic should be replaced by AvaliGraphic, or whatever you may change it to. It took a while to change it all, but it does work.